build-common.xml

```diff
@@ -1077,6 +1077,42 @@ information.
 		</sequential>
 	</macrodef>
 
+	<macrodef name="generate-pom-info">
+		<sequential>
+			<property name="artifact.name" value="${ant.project.name}" />
+
+			<if>
+				<available file="../tools/maven/${artifact.name}.xml" />
+
+				<then>
+					<xmlproperty file="../tools/maven/${artifact.name}.xml" />
+
+					<property file="bnd.bnd" />
```

Comment:
Instead of loading all properties, IIRC you can limit which property you want. @petershin, am I right?

Reply:
@Ithildir - Yup, you can use `loadproperties` with a nested `filterchain`.  Will include this info in the new review